### ISSUE TITLE
Ideas to improve the training accuracy

### ISSUE THREAD

**Issue by Leslie-Fang** [deepmind/kinetics-i3d#93]
I am trying to fine-tune the model with UCF-101 dataset.
with the SGD optimizer and lr decay, I can get almost 91.79% accuracy after 6000 steps with BS: 32.
```
global_step = tf.Variable(0, trainable=False)
learning_rate = tf.compat.v1.train.exponential_decay(3, global_step, 100, 0.96)
optimizer = tf.compat.v1.train.GradientDescentOptimizer(learning_rate).minimize(loss, global_step=global_step)
```
From the paper, I see the accuracy could be 94.x%, any ideas to improve my accuracy?

**Comment by joaoluiscarreira** [comment]
How many frames are being used in training and testing ?

Joao

**Comment by Leslie-Fang** [comment]
@joaoluiscarreira  Thanks for looking into my issue.
Training: almost 9500 frame in one epoch.
Testing: almost 3700 frame.
BTW: I am using depth 64.

**Comment by joaoluiscarreira** [comment]
Let me clarify: what we did was to use 64 frames in training and 250 frames
in testing. If you're testing on only 64 frames for each video that would
have a negative impact.

Joao

**Comment by jzq0102** [comment]
> 
> 
> 我正在尝试使用 UCF-101 数据集微调模型。 使用 SGD 优化器和 lr 衰减，在 BS: 32 的 6000 步后，我可以获得几乎 91.79% 的准确率。
> 
> ```
> global_step = tf.Variable(0, trainable=False)
> learning_rate = tf.compat.v1.train.exponential_decay(3, global_step, 100, 0.96)
> optimizer = tf.compat.v1.train.GradientDescentOptimizer(learning_rate).minimize(loss, global_step=global_step)
> ```
> 
> 从论文中，我看到准确率可能是 94.x%，有什么想法可以提高我的准确率吗？

Can I share the code？
